src/unit1/lesson7/README.md

```diff
@@ -231,31 +265,34 @@ private void Run()
             default:
                 return;
         }
-        Console.WriteLine("------------");
+
+        "------------".Display();
     }
 }
 ````
-### Step 6: Implementing the logic to create a new contact
+### Step 7: Implementing the logic to create a new contact
 
 ````csharp
 private void CreateContact()
 {
     using (var session = DocumentStoreHolder.Store.OpenSession())
     {
-        Console.WriteLine("Name: ");
-        var name = Console.ReadLine();
+        "Name: ".Display();
```

Comment:
I don't like this. `Console.WriteLine` is something that the reader is familiar with.
The aesthetics to be applied here call for familiarity and lack of surprise above all else.

Reply:
So, If the better to be used here is to keep `Console.WriteLine` for better readability to anyone reader, I think that class extension doesn't make sense to keep existing anymore, right?

Reply:
Yes

Reply:
It was rolled back to the original implementation using Console.WriteLine